cypress/e2e/cloud/about.test.ts

```diff
@@ -80,12 +90,17 @@ describe('About Page for PAYG users', () => {
   beforeEach(() =>
     cy.flush().then(() =>
       cy.signin().then(() => {
-        cy.get('@org').then(({id}: Organization) => {
-          cy.quartzProvision({
-            accountType: 'pay_as_you_go',
-          }).then(() => {
-            cy.visit(`/orgs/${id}/org-settings`)
-            cy.getByTestID('about-page--header').should('be.visible')
+        cy.setFeatureFlags({
```

Comment:
Need this separately in each `beforeEach` because they are part of separate `describe` blocks.

Reply:
Note that I've had to adjust the position of where we call `setFeatureFlags`. Certain pages we're navigating to can be 'flaky' with the flag reset, so in each case I've tested for the best location to reset flags (which isn't always identical between tests).